includes/Router.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace WC\SmoothGenerator;
+
+use WC\SmoothGenerator\Generator;
+
```

Comment:
```suggestion
```

We're already inside `WC\SmoothGenerator` and so can reference `Generator` without importing or qualifying it.

Reply:
...Or if the goal is to avoid confusion with PHP's Generator class, we could perhaps give it an alias. I don't think we need that, though, fwiw.

Reply:
ec02dce